jquery.unobtrusive-ajax.js

```diff
@@ -91,7 +91,7 @@
         $.extend(options, {
             type: element.getAttribute("data-ajax-method") || undefined,
             url: element.getAttribute("data-ajax-url") || undefined,
-            cache: !!element.getAttribute("data-ajax-cache"),
+            cache: (element.getAttribute("data-ajax-cache") || "").toLowerCase() === "true",
```

Comment:
I think if we wanted this to be supported it would need to be:

```javascript
function getBool(value, defaultValue)
{
    if (value === true) return true;
    if (value === 'true') return true;
    if (value === false) return false;
    if (value === 'false') return false;
    return defaultValue || false;
}
```

It doesn't work with non-lowercase strings with `true`/`false`, but I'm not sure that's much of an issue.

Then this code would call it like this:

```javascript
cache: getBool(element.getAttribute("data-ajax-cache"), /* defaultValue */ false), 
```

However, I think that working the DOM elements and attribute values expects that it's already a string, no? Setting it to a Boolean value doesn't sound right to me. Is that common in JS libraries?